exsrv.cmd

```diff
@@ -0,0 +1,71 @@
+:: Utility Batch File To Start/Stop Elixir App As Windows Service
+:: Onorio Catenacci
+:: 19 November 2015
+
+@echo off
+SET THISFILENAME=%~n0
+SET COMMAND=%2
+SET ERTS_BIN_PATH=C:/progra~1/erl7.1/erts-7.1/bin
```

Comment:
It's not the purpose of exrm to avoid having to install anything, though it is certainly one of the benefits, anyway I think Onorio left this in during testing and forgot to pull it out. It should work the same as the shell script, which has the runtime relative to the release directory when `include_erts` is set to something other than false in `relx.config`.